apps/style/common.scss

```diff
@@ -385,6 +385,10 @@ html[dir='rtl'] #codeWorkspace:not(.applab) {
   line-height: 400px;
 }
 
+html[dir='rtl'] div#visualizationResizeBar {
+  right: 400px;
+}
+
```

Comment:
On a similar note, at some point I'd like to replace all these 400px with a variable (that maybe also ends up in JS).